### DISCUSSION TITLE
Use initials instead of silhouette for users who have not uploaded a photo avatar

### DISCUSSION THREAD

**Discussion by aaronbushnell** [craftcms/cms#13492] [Ideas] [answered]
Perhaps it's just a taste thing, but I think using initials (though pulling this might be difficult in some cases) makes for a better default avatar when a user hasn't uploaded a photo.

Here's a before/after:
![Avatars](https://github.com/craftcms/cms/assets/315202/776465c3-774e-403d-a29e-560733e6fe4c)

The empty silhouette for every user can be difficult to scan and differentiate between the users in the CMS. I think using initials (and perhaps even differing colors) makes it easier to scan through the different authors in content views.

**Comment by aaronbushnell** [comment]
The addition of color is also nice (if possible) to make scanning these rows considerably easier. I know color alone isn't a good UX for users with color blindness, but I'm hoping the combination of the initials and color can make this a good experience.
![color-changes](https://github.com/craftcms/cms/assets/315202/61c6f938-b207-4fdb-a8fd-922ab74e2a67)

**Comment by brandonkelly** [comment, marked as answer]
Good idea! Was actually already thinking about doing this, to improve the new author notifications coming in 4.5 (#13420), which were pretty awkward for users without photos.

Done now, for 4.5: #13511

<img width="213" alt="Four users, with the new initials-based thumbnails" src="https://github.com/craftcms/cms/assets/47792/24dbb724-c009-4a26-a859-c46b14db23cd">

**Reply by aaronbushnell** [reply]
Wow, I _love_ this! Great work, @brandonkelly! ❤️

**Reply by brandonkelly** [reply]
Craft 4.5.0 is out with that change. Thanks for the suggestion!

**Reply by aaronbushnell** [reply]
Absolutely! Looks fantastic, @brandonkelly. Great work on this and the other 4.5 updates! 🎉

**Reply by brandonkelly** [reply]
Thanks :)

**Comment by shifuma** [comment]
This is cool! Is it possible to use on the front-end?

**Reply by brandonkelly** [reply]
No, it’s coded specifically for the control panel. You could come up with a similar effect though.
